gap/Magic.gd

```diff
@@ -303,6 +303,17 @@
 #!             <Br/>
 #!             <E>Default value: <C>[ ".", "gap", "lib", "examples", "examples/doc" ]</C>.</E>
 #!         </Item>
+#!         <Mark><A>gap_root_relative_path</A></Mark>
+#!         <Item>
+#!             Either true or a string representing a relative path. &GAPDoc;s MakeGAPDocDoc,
```

Comment:
It could also be set to `false`, couldn't it? How about: `Either a boolean or a string...`

Also, `&GAPDoc;s MakeGAPDocDoc,` should use `<C>` -- and I am not sure whether I'd mention it here at all...